recipes/apispec/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "apispec" %}
+{% set version = "0.16.0" %}
+{% set sha256 = "ecd0fc3d97dfcd7f643304cd5cc9d9d0b0964cf3d10514f0818bbc3dbe41820d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyyaml >=3.10
```

Comment:
Please drop anything from `build` that is not strictly a requirement at build time. Probably just need `python` and `setuptools`.